src/main/charts/jira/Changelog.md

```diff
@@ -1,4 +1,16 @@
 # Change Log
+
+## 1.6.0
+
+**Release date:** 2022-10-11
+
+![AppVersion: 8.20.13](https://img.shields.io/static/v1?label=AppVersion&message=8.20.13&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* Ingress Class Name is moved under spec. (#450)
+* Improved documentation (#448, #440)
+
 ## 1.5.1
```

Comment:
We didn't release 1.5.1. Maybe consolidate these two?

Reply:
These items are related to changes after 1.5.0 up to now.

Reply:
yes, what I meant is including the item on line 22 as well, because we never released 1.5.1 for Jira, and it was a placeholder for the next release.